jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/internal/HTTP2Flusher.java

```diff
@@ -294,7 +294,7 @@ protected Action process() throws Throwable
     }
 
     @Override
-    public void succeeded()
+    protected void onSuccess()
```

Comment:
I think this change alone might fix a lot of strangeness we get with failures!

Reply:
:+1: